modules/getting-started/proc-techdocs-configure-odf-helm.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc-techdocs-configure-odf-helm_{context}"]
+
+= Making object storage accessible to containers using the Helm chart
+
+`ObjectBucket` information is stored in the {product-short} `ObjectBucketClaim` config map and secret. Adding the `ObjectBucket` information to the Helm chart configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
+
+* `BUCKET_NAME`
+* `BUCKET_HOST`
+* `BUCKET_PORT`
+* `BUCKET_REGION`
+* `BUCKET_SUBREGION`
+* `AWS_ACCESS_KEY_ID`
+* `AWS_SECRET_ACCESS_KEY`
+
+These variables are then used in the TechDocs plugin configuration.
+
+.Prerequisites
+
+* You have installed {product} on {ocp-short} using the Helm chart.
+* You have created an `ObjectBucketClaim` CR for storing files generated by TechDocs.
```

Comment:
```suggestion
* You have created an `ObjectBucketClaim` custom resource (CR) for storing files generated by TechDocs.
```
I'd spell out what CR is for the first use in a module, however, this reads a little confusing to me considering the intro line - which CR are we referring to, the config map, or the secret, both, or a different CR entirely? Do we have docs maybe we can link to about creating this CR that would make this clearer?